File: Number Series/Basic/prog5.c
#include <stdio.h>

void main() {
	int row;
	printf("Enter rows: ");
	scanf("%d",&row);

	for(int olc=0; olc<row; olc++){
		for(int ilc=0; ilc<=olc; ilc++)
			printf(" * ");
		printf("\n")
	}

}

File: Number Series/Basic/prog5.java
import java.io.*;
import java.lang.System.*;

class Program {
	public static void main(String args[]) {

		BufferedReader br = new BuferedReader (new InputStreamReader(System.in));

		System.out.println("Enter row ")
		int row = Integer.parseInt(br.readLine());
		
		for(int olc =0 ; olc<row ;olc++) {
				for(int ilc = 1; ilc<= olc; ilc++)	
						System.out.print(" * ");
				System.out.println();
		}

		

	}
}


